interview-questions.md

```diff
@@ -7,62 +7,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: In my own words, a conditional statement is a way to imbue your code with logic. They let you perform actions based on whether certain conditions are met.
 
-  Researched answer:
+  Researched answer: W3 SCHOOLS DEFINITION: Conditional statements are used to perform different actions based on different conditions.
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: In my own words, GIT is an open source versioning tool used by developers to track changes in your code base.
 
-  Researched answer:
+  Researched answer: www.git.scm answer: Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: If I remember correctly, the JavaScript operators that return Boolean values are those operators that deal with comparison operations such as ==,===,>,< and a few others.
 
-  Researched answer:
+  Researched answer: W3Schools Answer: Comparison AND Logical operators are used to test for true or false.
```

Comment:
For sure.  While I like that you are citing your sources, just keep in mind that this should still be written in your own words.